src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -612,6 +612,7 @@ def _validate_metadata(
         requirements.lockfile_hex_digest,
         request.interpreter_constraints,
         python_setup.interpreter_universe,
+        {},  # TODO(chrisjrn): include requirements strings here
```

Comment:
At the moment I'm not passing the actual requirements strings in, because we're only validating for tool lockfiles. This will change once we start using user lockfiles (which themselves send down requirements strings)